charts/reporting-operator/templates/custom-resources/report-queries/pod-volume-request.yaml

```diff
@@ -165,3 +165,48 @@ spec:
     AND "timestamp" < timestamp '{| default .Report.ReportingEnd .Report.Inputs.ReportingEnd | prestoTimestamp |}'
     GROUP BY namespace
     ORDER BY volume_request_storage_byte_seconds DESC
+
+---
+
+apiVersion: metering.openshift.io/v1alpha1
+kind: ReportGenerationQuery
+metadata:
+  name: "cluster-persistentvolumeclaim-request"
+  labels:
+    operator-metering: "true"
+{{- block "extraMetadata" . }}
+{{- end }}
+spec:
+  reportQueries:
+  - "persistentvolumeclaim-request-raw"
+  view:
+    disabled: true
+  columns:
+  - name: period_start
+    type: timestamp
+    unit: date
+  - name: period_end
+    type: timestamp
+    unit: date
+  - name: data_start
+    type: timestamp
+    unit: date
+  - name: data_end
+    type: timestamp
+    unit: date
+  - name: volume_request_storage_bytes_seconds
```

Comment:
`bytes` should be `byte` here.

Reply:
Fixed in commit 8c15b524ce065275b661be4f40cab78955e1dab8